src/main/java/com/iwaconsolti/league/manager/controller/LeaguesController.java

```diff
@@ -6,13 +6,12 @@
 import com.iwaconsolti.league.manager.service.ILeagues;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.context.annotation.Profile;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+
 import java.util.List;
 
```

Comment:
In the LeaguesController constructor, the parameter names soccerLeague and baseBallLeague do not match the attribute names soccerLeagues and baseBallLeague. While this works, it is recommended that variable names be consistent for readability.